app/controllers/complaints_controller.rb

```diff
@@ -12,27 +12,13 @@ def show
     @complaint = Complaint.new(
       ApiDelegator.use("hses", "issue", {id: params[:id]}).request[:data]
     )
-    @issue_tta_reports = IssueTtaReport.where(issue_id: params[:id])
-    @timeline = Timeline.new(@complaint.attributes, tta_reports)
+    @issue_tta_reports = IssueTtaReport.where(issue_id: params[:id]).order(:start_date)
+    @timeline = Timeline.new(@complaint.attributes, @issue_tta_reports)
```

Comment:
[thought/question] hmm what were your thoughts on getting the description/topics/etc once we include those elements? were you envisioning bringing back the API call to gather those details if the user has permission?

Reply:
We already have the plumbing for getting the details of an activity report within `IssueTtaReport` so I was going to have it be responsible for showing all the fields.

Some details (like exactly how errors will be handled) are still a little fuzzy in my mind, but that's the big picture